source/FreeRTOS_TCP_IP_IPv6.c

```diff
@@ -100,6 +100,15 @@ BaseType_t xProcessReceivedTCPPacket_IPV6( NetworkBufferDescriptor_t * pxDescrip
 {
     /* Function might modify the parameter. */
     NetworkBufferDescriptor_t * pxNetworkBuffer = pxDescriptor;
+    const ProtocolHeaders_t * pxProtocolHeaders;
```

Comment:
Good cleanup, was it coming as a warning or MISRA error?

Reply:
TY, its part of the C90 compliance changes not MISRA.